.gitlab/release.yml

```diff
@@ -0,0 +1,19 @@
+release_ruby_gem:
+  stage: release
+  image: $DOCKER_REGISTRY/images/mirror/ruby:3.2.2  
+  tags: ["arch:amd64"]
+  only:
+    # v2.10.0
+    # v2.10.1
+    # v2.10.0rc0
+    # v2.10.0rc5
+    - /^v[0-9]+\.[0-9]+\.[0-9]+(rc[0-9]+)?$/
```

Comment:
```suggestion
    # Version pattern from Rubygems: https://github.com/rubygems/rubygems/blob/969f32050f0dc024c411fef7cbe25b1aaed16983/lib/rubygems/version.rb#L158
    # with a added `v` prefix.
    - /^v[0-9]+(?>\.[0-9a-zA-Z]+)*(-[0-9A-Za-z-]+(\.[0-9A-Za-z-]+)*)?$/
```

Reply:
Ruby has a slightly different version pattern, allowing alphanumeric characters in all positions, as well as separating the `rc` part with a `.`.